CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## v1.7.1 (November 9th, 2023)
+* chore: support hyphens for bug name regexp (#18)
```

Comment:
Unless we plan to do a release today (which I wasn't thinking we would do), I'd use "In development" as the date.

Reply:
Also, this file only uses "bug:" and "feature:" so far. I'm not sure what "chore" means.